docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md

```diff
@@ -15,39 +15,30 @@ Hardware requirements:
 <a name="on_Windows"></a>
 ## Installing Intel&reg; HAXM on Windows&reg;
 
-Supported Windows&reg; versions: 8.1/8/7 (32/64-bit), Vista (32/64-bit), XP SP2 or higher (32-bit only)
+Supported Windows&reg; versions: 10/8.1/8/7 (32/64-bit), Vista (32/64-bit), XP SP2 or higher (32-bit only)
 
-**HAXM is installed automatically as part of the Tizen Studio**. If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/6.0.4/win/](http://download.tizen.org/sdk/haxm/6.0.4/win/) and download the Windows installer package.
+**HAXM is installed automatically as part of the Tizen Studio**. If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/7.0.0/win/](http://download.tizen.org/sdk/haxm/7.0.0/win/) and download the Windows installer package.
 
 **Table: Windows installer package**
 
-| Windows installer                        | File description | File size | MD5 checksum                     |
+| Windows installer                        | File description | File size | SHA256SUM                     |
 |----------------------------------------|----------------|---------|--------------------------------|
-| [IntelHAXM_6.0.4.exe](http://download.tizen.org/sdk/haxm/6.0.4/win/IntelHAXM_6.0.4.exe) | System driver    | 2.66 MB   | bc05f881d9282ed0914b3e0bf3c96e89 |
+| [IntelHAXM_7.0.0.exe](http://download.tizen.org/sdk/haxm/7.0.0/win/IntelHAXM_7.0.0.exe) | System driver    | 2.89 MB   | 1587f7ae42e11c7b22dc674fd776ce252b6a19152e7ca416a748e2bdbf2b36c5 |
```

Comment:
In my opinion there is no point in giving the same link twice in the same section. 
Please remove the URL from the table and keep the filename in BOLD. 
You can give the full path to the URL in the previous paragraph.